libstuff/ResourceMonitorThread.h

```diff
@@ -0,0 +1,27 @@
+#include "libstuff/libstuff.h"
+#include <thread>
+
+using namespace std;
+
+class ResourceMonitorThread: public thread
+{
+    ResourceMonitorThread(const ResourceMonitorThread&) = delete;
+    ResourceMonitorThread(ResourceMonitorThread&&) = delete;
```

Comment:
These two lines should be able to be removed, they were for testing.

Reply:
Removed!